com.chartboost.mediation/Runtime/Plugins/iOS/ChartboostMediationUnityBridge.mm

```diff
@@ -603,6 +603,38 @@ void _chartboostMediationDiscardOversizedAds(BOOL shouldDiscard)
     [[Helium sharedHelium] setDiscardOversizedAds:shouldDiscard];
 }
 
+const char * _chartboostMediationInitializedAdaptersInfo() {
+    
+    NSMutableArray * jsonArray = [NSMutableArray array];
+    
+    NSArray<HeliumAdapterInfo*> * adapters =  [[Helium sharedHelium] initializedAdapterInfo];
+    for (HeliumAdapterInfo *adapter in adapters) {
+        NSString * partnerVersionKey        = @"partnerVersion";      NSString * partnerVersionValue     = adapter.partnerVersion;
+        NSString * adapterVersionKey        = @"adapterVersion";      NSString * adapterVersionValue     = adapter.adapterVersion;
+        NSString * partnerIdentifierKey     = @"partnerIdentifier";   NSString * partnerIdentifierValue  = adapter.partnerIdentifier;
+        NSString * partnerDisplayNameKey    = @"partnerDisplayName";  NSString * partnerDisplayNameValue = adapter.partnerDisplayName;
+
+        NSDictionary *adapterDictionary = [NSDictionary dictionaryWithObjectsAndKeys:
+          partnerVersionValue,partnerVersionKey,
+          adapterVersionValue,adapterVersionKey,
+          partnerIdentifierValue,partnerIdentifierKey,
+          partnerDisplayNameValue,partnerDisplayNameKey,
+          nil
+        ];
+    
+        [jsonArray addObject:adapterDictionary];
+    }
+    
+    NSError *error;
+    NSData *jsonData = [NSJSONSerialization dataWithJSONObject:jsonArray options:0 error:&error];
+    if (! jsonData) {
+        NSLog(@"%s: error: %@", __func__, error.localizedDescription);
+        return "";
+     }
+    NSString *json = [[NSString alloc] initWithData:jsonData encoding:NSUTF8StringEncoding];
```

Comment:
I believe we have a Utility function somewhere to convert any NSDictionary into a JSON Object. We do this for Core 🤔 and I believe I grabbed it from Mediation.

Reply:
yeah the utility function was to convert a `NSDictionary` but in this case we have an array of `NSDictionary`. The underlying code to convert to json is same for array or dictionary so I combined them into one function `toJSON`